src/cdn_definitions/schema.json

```diff
@@ -44,6 +44,22 @@
             },
             "type": "object"
         },
+        "content_set_label": {
+            "pattern": "^[A-Za-z0-9_\\-]{1,100}$",
```

Comment:
I'm just wondering if 100 chars might be a bit short?  I checked our current data and the current longest label seems to be 74 chars already: ansible-automation-platform-2.0-early-access-for-rhel-8-x86_64-source-rpms

So, that's not much room to grow.

When I'm in doubt about upper limits I usually set them to at least 2x the time of the currently known max.

Reply:
update the cs pattern to:
` "pattern": "^[A-Za-z0-9_\\-]{1,200}$",`